drama-queen/src/ui/components/orchestrator/buttons/PrevNext/PrevNext.tsx

```diff
@@ -23,14 +24,16 @@ export function PrevNext(props: PrevNextProps) {
     isLastPage,
     isLastReachedPage,
     readonly,
+    hasPageResponse,
     goPrevious,
     goNext,
   } = props
   const { classes, cx } = useStyles()
   const { t } = useTranslation('navigationMessage')
 
-  // we always display nextButton on readonly mode
-  const canGoNext = (!isLastReachedPage || readonly) && !isLastPage
+  const canGoNext =
+    ((!isLastReachedPage && hasPageResponse()) || readonly) && !isLastPage
+
```

Comment:
What do you think about handle this case in `useContinueBehavior` ? 

A potential solution is to modify useContinueBehavior to return { label: undefined, onContinue: undefined } in specific scenarios. This approach ensures that the UI component remains agnostic to business logic changes, leading to a more maintainable and scalable codebase.


```tsx
- {canGoNext && (
-            <ShortCut shortCutKey={nextShortCutKey} onClickMethod={goNext} />
-          )}
+ {goNext && <ShortCut shortCutKey={nextShortCutKey} onClickMethod={goNext} />}
          
```